package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "cdmx009-mdlinks",
+  "version": "1.0.0",
+  "description": "Libreria de node para archivos md",
```

Implementa tu descripción en ingles
```suggestion
  "description": "Library with node for read md files",
```